mokapot/model.py

```diff
@@ -564,14 +591,15 @@ def _get_starting_labels(psms, model):
 
     elif model.is_trained:
```

Comment:
We still need to update the logic here. If our pre-trained model does not yield as many PSMs as the best feature, the best feature should be used and a warning message should be issued.